wandb/sdk_py27/wandb_init.py

```diff
@@ -546,13 +584,15 @@ def init(self):  # noqa: C901
 def getcaller():
     # py2 doesnt have stack_info
     # src, line, func, stack = logger.findCaller(stack_info=True)
-    src, line, func = logger.findCaller()[:3]
-    print("Problem at:", src, line, func)
+    # src, line, func = logger.findCaller()[:3]
+    # print("Problem at:", src, line, func)
+    # traceback.print_exc()
+    pass
```

Comment:
Should we just get rid of this, or we want to keep it for our own debugging scenarios?